docs/sequence/PISPPreLinkingApi.puml

```diff
@@ -0,0 +1,155 @@
+@startuml
+
+title PISP Pre-Linking
+
+participant "PISP Backend" as PISP
+box "PISP tp-scheme-adapter"
+  participant "outbound-server" as PISP_TP_OUT
+  participant "PISPrelinkingModel" as PISP_PLM
+  participant "inbound-server" as PISP_TP_IN
+end box
+box "Mojaloop"
+    participant Switch
+end box
+
+== Pre-Linking ==
+autonumber 1 "<b>PRELINKING-#</b>"
+
+PISP -> PISP_TP_OUT: GET /linking/get-dfsps
+
+activate PISP
+activate PISP_TP_OUT
+
+PISP_TP_OUT -> PISP_TP_OUT: const model = await create()
+rnote right of PISP_TP_OUT: state: start
+PISP_TP_OUT -> PISP_PLM: model.getThirdpartyEnabledDFSPs()
+
+activate PISP_PLM
+
+PISP_PLM -> PISP_PLM: ThirdpartyRequests.getServicesById()
+PISP_PLM -> Switch: GET /services/THIRD_PARTY_DFSP
+
+activate Switch
+Switch --> PISP_PLM: 202 Accepted
+deactivate PISP_PLM
+
+note over PISP, Switch
+  HAPPY_SCENARIO: Switch returns a non-empty list
+end note
+
+activate PISP_TP_IN
+
+Switch --> PISP_TP_IN: PUT /services/THIRD_PARTY_DFSP
+rnote left of Switch #LightBlue
+PUT /services/THIRD_PARTY_DFSP
+{
+  serviceProviders: [
```

Comment:
```suggestion
   providers: [
```

Reply:
https://github.com/mojaloop/pisp/blob/master/docs/linking/README.md#142-otp has it as `serviceProviders`. Was this changed somewhere else?

makes sense though a bit redundant to have "service" repeated on the services resource.